src/fsharp/TypedTree.fs

```diff
@@ -39,9 +39,6 @@ type StampMap<'T> = Map<Stamp, 'T>
 [<RequireQualifiedAccess>]
 type ValInline =
 
-    /// Indicates the value must always be inlined and no .NET IL code is generated for the value/function
```

Comment:
I guess we don't need this anymore? Seems like a good thing. :)

Reply:
Right, we could factor it out as separate cleanup